quickwit/quickwit-serve/src/search_api/rest_handler.rs

```diff
@@ -232,19 +291,21 @@ pub fn search_stream_handler(
 
 /// This struct represents the search stream query passed to
 /// the REST API.
-#[derive(Deserialize, Debug, Eq, PartialEq)]
+#[derive(Deserialize, Debug, Eq, PartialEq, utoipa::IntoParams)]
+#[into_params(parameter_in = Query)]
 #[serde(deny_unknown_fields)]
 struct SearchStreamRequestQueryString {
     /// Query text. The query language is that of tantivy.
     pub query: String,
     // Fields to search on.
+    #[param(rename = "search_field")]
     #[serde(default)]
     #[serde(rename(deserialize = "search_field"))]
     #[serde(deserialize_with = "from_simple_list")]
     pub search_fields: Option<Vec<String>>,
     /// Fields to extract snippet on
     #[serde(default)]
-    #[serde(rename(deserialize = "snippet_fields"))]
+    #[serde(rename(deserialize = "snippet_fields"))] // TODO: Was this supposed to be `snippet_field`? - CF
```

Comment:
Random thing I noticed when going through the models, this is renamed to the same things that it already is, was this supposed to be `snippet_field` and can this be removed? @fmassot

Reply:
Maybe @evanxg852000 knows that